uportal-war/src/main/resources/org/jasig/portal/portlets/SharedParameters.cpd.xml

```diff
@@ -33,54 +33,40 @@
         <parameter>
             <name>iconUrl</name>
             <label>display.icon.url</label>
+            <description>URL of the icon to use in the standard portlet</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
             <name>mobileIconUrl</name>
             <label>display.mobile.icon.url</label>
+            <description>URL of the icon to use on Mobile devices</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
-            <name>showChrome</name>
-            <label>show.portlet.window.chrome</label>
-            <single-choice-parameter-input display="select">
-                <default>true</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
-
-        <parameter>
             <name>alternativeMaximixedLink</name>
             <label>open.alternative.url.in.maximized.view</label>
+            <description>Alternate URL to display when the portlet is in MAXIMIZED mode.</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
-            <name>alternate</name>
-            <label>use.alternate.appearance</label>
-            <single-choice-parameter-input display="select">
-                <default>false</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
```

Comment:
Oh.  Good catch.  Need to add an additional "when" to check the "chromeStyle" param to the code that sets the PORTLET_ALTERNATE variable.